pallets/airdrop/src/benchmarking.rs

```diff
@@ -37,12 +38,33 @@ benchmarks! {
         <T as pallet_gear::Config>::Currency::deposit_creating(&source, (1u128 << 60).unique_saturated_into());
         let recipient: T::AccountId = benchmarking::account("recipient", 0, 0);
         // Keeping in mind the existential deposit
-        let amount = 100_000_u128.saturating_add(10_u128.saturating_mul(q.into()));
+        let amount = 10_000_000_000_000_u128.saturating_add(10_u128.saturating_mul(q.into()));
 
     }: _(RawOrigin::Root, source, recipient.clone(), amount.unique_saturated_into())
     verify {
         assert_eq!(pallet_balances::Pallet::<T>::total_balance(&recipient), amount.unique_saturated_into());
     }
+
+    transfer_vested {
+        let q in 1 .. 256;
```

Comment:
I'm not quite getting the meaning of the complexity parameter `q` here: seems it only affects the amount being transferred. Off the top of my head the resulting weight shouldn't depend on it so we could have a constant weight function for this call. Or am I mistaken?

Reply:
Yeah you're right. But after I implemented benchmarks I remembered about sudo call😅